deploy/pagerduty-service-orchestration/00-osd-serviceorchestration-configmap.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+data:
+  service-orchestration.json: |
+    {
+    "orchestration_path": {
+    "catch_all": {
+        "actions": {}
+    },
+    "sets": [
+    {
+        "id": "start",
+        "rules": [
+        {     
+            "actions": {
+            "suppress": true
+            },
+            "conditions": [
+            {
+            "expression": "event.severity matches 'Warning'"
+            }
+            ]
+        }
+        ]
+    }
+    ],
+    "type": "service"
+    }
+    }
+kind: ConfigMap
+metadata:
```

Comment:
The PD operator will read the configmap which has the same label as the clusterdeployment, for filter.
So in here, I think we should have the label which indicates the cluster is managed as well.
```
metadata:
  labels:
    api.openshift.com/managed: "true"

Reply:
Thanks for the review @bmeng . I have addressed this in my latest commit